templates/index.html

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+    <title> Index of {{ path }} </title>
+    <link rel="stylesheet" href="{{url_for('static', filename='/common/css/fileindex.css')}}" type="text/css">
+</head>
+<body>
+    <h1> Index of {{ path }} </h1>
+    <table id="indexlist">
```

Comment:
Maybe we can make the code nicer like [there](https://stackoverflow.com/a/62838219)

Reply:
Done. See the last patch.